src/resolvers/ens.ts

```diff
@@ -10,9 +10,9 @@ export default async function resolve(name: string) {
       return false;
     }
 
-    const url = await ensResolver.getText('avatar');
+    let url = await ensResolver.getText('avatar');
     if (!url || !url.startsWith('http')) {
-      return false;
+      url = `https://metadata.ens.domains/mainnet/avatar/${name}`;
     }
```

Comment:
Not important but there is a solarlint notice (I use) 
```
Prefer using an optional chain expression instead, as it's more concise and easier to read.sonarlint(typescript:S6582)
```
Maybe we can try this
```
url = url?.startsWith('http') ? url : `https://metadata.ens.domains/mainnet/avatar/${name}`;
```

Reply:
Done